auto_nag/scripts/leave_open_no_activity.py

```diff
@@ -0,0 +1,73 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.people import People
+from auto_nag import utils
+
+
+class LeaveOpenNoActivity(BzCleaner):
+    def __init__(self):
+        super(LeaveOpenNoActivity, self).__init__()
+        self.people = People()
+        self.nmonths = utils.get_config(self.name(), 'months_lookup', 6)
```

Comment:
Can you remove the 6 as we have that in the config file?
I found that confusing.

Reply:
It's just the default value

Reply:
I know but what is the point as it is defined in the conf?

Reply:
ok